ultimatepython/advanced/file_handling.py

```diff
@@ -0,0 +1,49 @@
+"""
+File handling is a fundamental concept in Python that involves opening, reading, writing, and appending to files.
```

Comment:
Does it make sense for us to explore this?

https://docs.python.org/3/library/tempfile.html

My concern is that this module will pollute the local directory/space with files that the developer / reader does not want to keep on his/her machine

Reply:
Can I add a delete file function to delete the file after completing all the operations. This will allow the developer to delete the file if they don't need it.

Reply:
Sure. Just make sure to check that the file no longer exists via an `assert` statement